lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -156,18 +184,16 @@ def write(chunk)
         target_index = record.delete @target_index_key
       elsif @logstash_format
         if record.has_key?("@timestamp")
-          time = Time.parse record["@timestamp"]
+          dt = @time_parser.call(record["@timestamp"]) rescue Time.at(time).to_datetime
```

Comment:
I meant fallback to ingestion time but log that you're doing it, don't drop the log.